server/src/routes/chats.routes.ts

```diff
@@ -0,0 +1,16 @@
+import { Router } from 'express';
+import {
+    createChat,
+    getChatById,
+    sendMessage
+} from '../controllers/chats.controller';
+import { isAuthenticated } from '../middleware/auth.middleware';
+
+const router = Router();
+
+router.get('/chat/:id',getChatById); //get chat based on chat id
```

Comment:
Should the user not require authentication before being able to view a chat?

Reply:
😬It should, I forgor about that :skull:

Reply:
Can you clearly fix that before I merge?